python/thunder/rdds/fileio/imagesloader.py

```diff
@@ -75,34 +76,82 @@ def fromStack(self, dataPath, dims, dtype='int16', ext='stack', startIdx=None, s
             If true, will recursively descend directories rooted at datapath, loading all files in the tree that
             have an extension matching 'ext'. Recursive loading is currently only implemented for local filesystems
             (not s3).
+
+        nplanes: positive integer, default None
+            If passed, will cause a single binary stack file to be subdivided into multiple time points. Every
```

Comment:
Let's avoid the explicit references to `time`. I just checked and we're generally good about this, though in fixing this would be great if you caught other users. There's certainly a notion of the images being /ordered/ both here and elsewhere, but that ordering need not relate to time specifically.